packages/hub/db/migrations/20211207151150639_sent-push-notifications.js

```diff
@@ -0,0 +1,23 @@
+const SENT_PUSH_NOTIFICATIONS_TABLE = 'sent_push_notifications';
+
+exports.up = (pgm) => {
+  // TODO: may be useful to add a message id field for debug purposes
+  // The message id is returned by firebase messaging
+  pgm.createTable(SENT_PUSH_NOTIFICATIONS_TABLE, {
+    transaction_hash: { type: 'string', notNull: true },
+    owner_address: { type: 'string', notNull: true },
+    push_client_id: { type: 'string', notNull: true },
+    notification_type: { type: 'string', notNull: true },
+    notification_title: { type: 'string' },
+    notification_body: { type: 'string', notNull: true },
+    notification_data: { type: 'json' },
+  });
+
+  pgm.createIndex(SENT_PUSH_NOTIFICATIONS_TABLE, ['transaction_hash', 'owner_address', 'push_client_id'], {
```

Comment:
Do you know if we're going to have any notifications that aren't tied to a transaction_hash in the future?

Reply:
Not that I know of, but maybe adding event type to the index and replacing transaction hash with `event_id` might be good just in case?

Reply:
This part of the design does seem odd to me.